mariadb/repo.sls

```diff
@@ -1,4 +1,6 @@
 {%- set lsb_codename = salt['grains.get']('lsb_distrib_codename') %}
+{%- set stable_version = '10.1' %}
```

Comment:
Kunnen we hier niet beter een map.jinja van maken? Dit kan in principe mss nog verschillen per omgeving ook.